Story/StoryHooks.cs

```diff
@@ -57,14 +58,67 @@ private void StoryHooks()
             On.SLOracleSwarmer.BitByPlayer += SLOracleSwarmer_BitByPlayer;
             On.CoralBrain.CoralNeuronSystem.PlaceSwarmers += OnCoralNeuronSystem_PlaceSwarmers;
             On.SSOracleSwarmer.NewRoom += SSOracleSwarmer_NewRoom;
+            On.HUD.TextPrompt.Update += TextPrompt_Update;
 
+            On.HUD.TextPrompt.UpdateGameOverString += TextPrompt_UpdateGameOverString;
+        }
+
+
+        private void TextPrompt_UpdateGameOverString(On.HUD.TextPrompt.orig_UpdateGameOverString orig, TextPrompt self, Options.ControlSetup.Preset controllerType)
+        {
+            if (OnlineManager.lobby.gameMode is StoryGameMode)
+            {
+                if (!OnlineManager.lobby.isOwner)
+                {
+                    self.gameOverString = "A slugcat has fallen. Perform a rescue, or wait for host to shelter or die";
+
+                }
+
+                if (OnlineManager.lobby.isOwner)
+                {
+                    self.gameOverString = "A slugcat has fallen. Perform a rescue, shelter, or press PAUSE BUTTON to restart";
+
+                }
+            }
+            else
+            {
+                orig(self, controllerType);
+
+            }
+
+        }
+
+        private void TextPrompt_Update(On.HUD.TextPrompt.orig_Update orig, TextPrompt self)
+        {
+            orig(self);
+            if (OnlineManager.lobby.gameMode is StoryGameMode)
+            {
+
+                if (!OnlineManager.lobby.isOwner && self.currentlyShowing == TextPrompt.InfoID.GameOver)
+                {
+                    self.restartNotAllowed = 1; // block clients from GoToDeathScreen
```

Comment:
Not sure how this blocks clients. It looks like it's only used to determine if the hud should play the game over sound.

Reply:
![image](https://github.com/user-attachments/assets/f08612f4-d6bd-46bf-a35c-cbe03dfb7120)

`restartNotAllowed `decrements during Update. When it hits 0 and a player inputs the Option / whatever button is mapped, they will be process switched to GameOver. So in order to provide the player the ability to be dead and still have access to the menu and prevent game over mode,  I have to block the timer.

`lastAnyKey `could be overwritten instead, but because it's assigned to if player touched input in Update() if not in death screen, we'd have to re-write more of Update() instead of just holding the timer at 1.